tests/integration/testlib/examplelib.py

```diff
@@ -52,7 +54,9 @@ def load_example(name):
     examples = find_examples_dir()
 
     with open(os.path.join(examples, name)) as yamlfile:
-        state = yaml.load(yamlfile, Loader=yaml.SafeLoader)
+        content = yamlfile.read()
+        content = content.replace('eth1', TEST_NIC1)
```

Comment:
Not sure why you have not changed the names in the example files, but I'm fine with either way.